File: Skybuddy_App/Application/app/src/main/java/com/example/jugid/skybuddy/Objects/Mark.java
package com.example.jugid.skybuddy.Objects;

import com.google.gson.annotations.SerializedName;

public class Mark {
    @SerializedName("idmark")
    private String idmark;
    @SerializedName("contenu")
    private String contenu;
    @SerializedName("valeur")
    private String valeur;
    @SerializedName("dateMark")
    private String dateMark;
    @SerializedName("idvols")
    private String idvols;
    @SerializedName("user_noteur")
    private String user_noteur;
    @SerializedName("user_note")
    private String user_note;

    public String getIdmark() {
        return idmark;
    }

    public String getContenu() {
        return contenu;
    }

    public String getValeur() {
        return valeur;
    }

    public String getDateMark() {
        return dateMark;
    }

    public String getIdvols() {
        return idvols;
    }

    public String getUser_noteur() {
        return user_noteur;
    }

    public String getUser_note() {
        return user_note;
    }

    public void setIdmark(String idmark) {
        this.idmark = idmark;
    }

    public void setContenu(String contenu) {
        this.contenu = contenu;
    }

    public void setValeur(String valeur) {
        this.valeur = valeur;
    }

    public void setDateMark(String dateMark) {
        this.dateMark = dateMark;
    }

    public void setIdvols(String idvols) {
        this.idvols = idvols;
    }

    public void setUser_noteur(String user_noteur) {
        this.user_noteur = user_noteur;
    }

    public void setUser_note(String user_note) {
        this.user_note = user_note;
    }
}

File: Skybuddy_Server/src/modeles/mark.js
class Mark {
    constructor (idmark,contenu,valeur,dateMark,idvols,user_noteur,user_note) {
      this.idmark = idmark;
      this.contenu =contenu;
      this.valeur = valeur;
      this.dateMark = dateMark;
      this.idvols = idvols;
      this.user_noteur = user_noteur;
      this.user_note = user_note;
    }
    toJson(){
      return {
        'idmark':this.idmark,
        'contenu':this.contenu,
        'valeur':this.valeur,
        'dateMark':this.dateMark,
        'idvols':this.idvols,
        'user_noteur':this.user_noteur,
        'user_note':this.user_note
      };
    }
  }

  module.exports = Mark;
